docs/eventing/samples/sequence/sequence-reply-to-event-display/README.md

```diff
@@ -0,0 +1,202 @@
+---
+title: "Sequence Wired to event-display"
+weight: 20
+type: "docs"
+---
+
+# Using Sequences in series
+
+## Prerequisites
+
+For this example, we'll assume you have set up an `InMemoryChannel`
+as well as Knative Serving (for our functions). The examples use `newbroker`
```

Comment:
what's wrong w/ `default` ?

Reply:
I feel like there's typically too many things that assume default namespace and some things don't work. Anyways, changed to default.